### ISSUE TITLE
[BUG] Rule socket-communication-00000 not matching correctly

### ISSUE THREAD

**Issue by jmle** [konveyor/analyzer-lsp#413]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Konveyor version

0.3.0-beta2

### Priority

Critical

### Current Behavior

The rule does not match and issues are not created. If the "PACKAGE" location is removed, the rule matches.

Related to https://github.com/konveyor/analyzer-lsp/issues/363

### Expected Behavior

The rule should match normally

### How Reproducible

Always (Default)

### Steps To Reproduce

Check https://github.com/konveyor/analyzer-lsp/issues/363 for steps on how to reproduce

### Environment

```markdown
Check https://github.com/konveyor/analyzer-lsp/issues/363 for environment
```

**Comment by jmle** [comment]
`PACKAGE` location does not seem to work as expected, probably all these locations should be switched with `IMPORT`... Created shim PR: https://github.com/konveyor/windup-shim/pull/103
